client/src/components/DisplayTable/DisplayTable.stories.tsx

```diff
@@ -48,13 +59,13 @@ const table_with_URL_column_configs = {
   ...common_column_configs,
   label: {
     ...common_column_configs.label,
-    formatter: (id) => (
+    formatter: (id: CellValue) => (
       <a
         href={`https://en.wikipedia.org/wiki/List_of_pizza_varieties_by_country#${id}`}
         target="_blank"
         rel="noopener noreferrer"
       >
-        {country_lookup[id]}
+        {country_lookup[id as Country]}
       </a>
```

Comment:
I think, other type issues asside, this could be `(id: Country) => (`. `Country` is just a union of strings, and `Strings` is a valid `CellValue` type.